plugins/inputs/docker/docker.go

```diff
@@ -141,6 +160,21 @@ func (d *Docker) Description() string {
 	return "Read metrics about docker containers"
 }
 
+// Config Validation
+func (d *Docker) Init() error {
+	err := choice.CheckSlice(d.PerDeviceInclude, containerMetricClasses)
+	if err != nil {
+		return fmt.Errorf("error validating 'perdevice_include' setting : %v", err)
+	}
+
+	err = choice.CheckSlice(d.TotalInclude, containerMetricClasses)
+	if err != nil {
+		return fmt.Errorf("error validating 'total_include' setting : %v", err)
+	}
+
```

Comment:
Is having both lists empty a valid option? If not, please check here and return an error.

Reply:
I would say it is; by having both lists empty there are still a couple of lines written by the plugin (memory, container stats...)

```
> docker,engine_host=myhost,host=myhost,server_version=19.03.12 n_containers=3i,n_containers_paused=0i,n_containers_running=0i,n_containers_stopped=3i,n_cpus=8i,n_goroutines=34i,n_images=68i,n_listener_events=0i,n_used_file_descriptors=21i 1612686961000000000
> docker,engine_host=myhost,host=myhost,server_version=19.03.12 memory_total=33616916480i 1612686961000000000
```